extensions/wolfram-alpha/package.json

```
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "wolfram-alpha",
+  "title": "Wolfram Alpha",
+  "description": "Query Wolfram Alpha right from Raycast",
+  "icon": "command-icon.png",
+  "author": "mathieudutour",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Query Wolfram Alpha",
+      "description": "Template for showing a static list and a detail view",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "appId",
+      "type": "textfield",
```

```suggestion
      "type": "password",
```
Could be a password?